bin/geni-sync-wireless

```diff
@@ -0,0 +1,618 @@
+            if not geni_project_info:
+                print "Removing group %s" % group_name
+                self._orb.delete_group(group_name)
+                self._deleted_orbit_groups.append(group_name)
+
+    # Disable users who are only in the ORBIT holdingpen group
+    # Note: we've deleted some projects at this point, so 
+    # we mean users who are in at least one recently deleted group
+    # but no other non-deleted groups
+    def disable_users_in_no_project(self): 
+        for orbit_username in self._orbit_users:
+            geni_username = self.to_geni_name(orbit_username)
+            user_in_some_deleted_group = False
+            user_in_some_non_deleted_group = False
+            for group_name, group_info in self._orbit_groups.items():
+                if group_name == self._options.holdingpen_group: continue
+                recently_deleted_from_group = \
+                    orbit_username in self._deleted_orbit_members and \
+                    group_name in self._deleted_orbit_members[orbit_username]
+                # Check of you've been deleted from a group
+                if orbit_username in group_info['users']:
+                    if group_name in self._deleted_orbit_groups  or \
+                            recently_deleted_from_group:
+                        user_in_some_deleted_group = True
+                # Check if you're still in a group
+                    else:
+                        user_in_some_non_deleted_group = True
+                        break
+            # If you're recently deleted from a group and not in another
+            # group, disable user
+            if user_in_some_deleted_group and \
+                    not user_in_some_non_deleted_group:
+                print "Disabling user: %s" % orbit_username
+                self._orb.disable_user(orbit_username)
+
+    # Lookup GENI project by name
+    def lookup_geni_project(self, project_name):
+        for project_id, project_info in self._geni_projects.items():
+            if project_info['project_name'] == project_name:
+                return  project_info
+        return None
+
+    # Grab project info [indexed by project id] for all wimax-enabled projects
+    # Only single project for --project option
+    # Only projects to which given users belongs for --user option
+    def get_geni_projects(self):
+        projects = {}
+
+        # Get all the WIMAX-enabled projects
+        query = self._session.query(self.PROJECT_TABLE.c.lead_id, 
+                                    self.PROJECT_TABLE.c.project_name, 
+                                    self.PROJECT_TABLE.c.project_id, 
+                                    self.PROJECT_TABLE.c.project_purpose) 
+        query = query.filter(self.PROJECT_TABLE.c.project_id == \
+                                 self.PROJECT_ATTRIBUTE_TABLE.c.project_id)
+        query = query.filter(self.PROJECT_ATTRIBUTE_TABLE.c.name == \
+                                 'enable_wimax')
```

Comment:
Essentially, we add the attribute (with 't') if enabling and then remove the attribute if disabling. There is no case where we care about the value, just the presence of the attribute.